sonic_platform_base/sonic_xcvr/xcvr_eeprom.py

```diff
@@ -54,7 +54,6 @@ def read_raw(self, offset, size, return_raw = False):
             data = struct.unpack("%dB" %size, raw_data)
       return data
 
-
```

Comment:
@prgeor Can you please revert the changes in this file?

Reply:
@mihirpat1 there were two new lines I removed one unnecessary new line. let me know if that is what you want.